util/validations.go

```diff
@@ -27,3 +30,17 @@ func IsDNSCompliant(name string) bool {
 	match := compiledRegex.Match([]byte(name))
 	return match
 }
+
+func ValidateCoOrdinates(latitude string, longitude string) error {
+	if strings.Contains(latitude, "e") || strings.Contains(longitude, "e") {
```

Comment:
We can check for any alphanumeric character.

Reply:
this doesn't matter because below 39, 40 float conversion will fail and webhook return the mesage

Reply:
this **e** is to validate the earlier level mistake. I think we can remove that

Reply:
I guess the `e` check is to invalidate float exponents. 
Eg: **4e-06**

Reply:
Regex is just a suggestion. It would be a bit faster than the individual checks. 
the manual validation is also fine, but we also add `E` along with `e` for the exponent check.